code/game/objects/items/plushes.dm

```diff
@@ -31,7 +31,16 @@
 	var/normal_desc
 	//--end of love :'(--
 
-/obj/item/toy/plush/Initialize()
+	var/snowflake_id				//if we set from a config snowflake plushie.
+	var/abstract_type = /obj/item/toy/plush		//to prevent us from spawning "base" plushies. set this to a plush's own type. only use this if a plush is not meant to be standalone/spawned, ever.
```

Comment:
This could as well be a boolean tbh, the random plushies list is made of **sub**types and no other blacklisted "base" plush has real subtypes rn.